src/libraries/Microsoft.PowerFx.Core/Functions/Blob/StreamBlob.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.IO;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    public class StreamBlob : ByteArrayBlob
+    {
+        public StreamBlob(Stream stream)
+            : base(FromStreamToBytes(stream))
+        {
+        }
+
+        private static byte[] FromStreamToBytes(Stream stream)
+        {
+            using var ms = new MemoryStream();
+            stream.CopyTo(ms);
```

Comment:
> stream.CopyTo(ms); [](http://example.com/codeflow?start=11&length=19)

We should probably have some limits here

Reply:
PVA seems to have a need up to 4TB
I recommended the use of URI where possible instead but we should probably allow it